distributed/shuffle/tests/test_shuffle.py

```diff
@@ -0,0 +1,137 @@
+import asyncio
```

Comment:
Once I switched the inputs on these tests to be fairly fast they started hanging sometimes.  I'm not yet sure why.